VideoApi/VideoApi.DAL/Commands/CreateConferenceCommand.cs

```diff
@@ -58,6 +64,24 @@ public async Task Handle(CreateConferenceCommand command)
             {
                 conference.AddParticipant(participant);
             }
```

Comment:
Any reason why this code block is not moved to another class, as its exactly the same in the AddParticipantsToConferenceCommand?